src/main/resources/swagger/thurloe.yaml

```diff
@@ -119,6 +119,17 @@ paths:
       security:
         - google_oauth:
             - openid
+  /thurloe/status:
+    get:
+      operationId: status
+      summary: Returns status code for connection to database.
+      produces:
+      - text/plain
```

Comment:
add a tag like "Status" so this can show up as a new category in swagger. something like: 

`tags:
  - Status`